lib/Sema/TypeCheckDeclPrimary.cpp

```diff
@@ -3213,9 +3213,9 @@ class DeclChecker : public DeclVisitor<DeclChecker> {
                   auto candidateType = candidate->getInterfaceType()->getRValueType();
                   if (candidateType->isEqual(declReturnType))
                     FD->diagnose(diag::override_method_in_lieu_of_property, declName.getIdentifier())
-                        .fixItReplace(FD->getSourceRange(),
-                                      "var " + declName.getIdentifier().str().str() + ": " + declReturnType.getString() + " { <#code#> }")
-                        .fixItRemove(FD->getBodySourceRange());
+                        .fixItRemove(FD->getBodySourceRange())
+                        .fixItReplace(FD->getSourceRangeIncludingAttrs(),
```

Comment:
```
error: property does not override any property from its superclass
  override var canBecomeFirstResponder: Int {
  ~~~~~~~~     ^
note: did you mean to override the method 'canBecomeFirstResponder'?
  override var canBecomeFirstResponder: Int {
               ^~~~~~~~~~~~~~~~~~~~~~~
               override func canBecomeFirstResponder() -> Int { <#code#> }
```

Reply:
In this case, the replacement starts after `override var`...

Reply:
I left a comment about this in previous fix-it location.